### PR TITLE
Update README.md with latest thoughtbot info and contribution info

### PR DESCRIPTION
**Pull request by jsom**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License
 
-Copyright (c) 2012 thoughtbot, inc.
+Copyright (c) 2015 thoughtbot, inc.
```
Comment (mike-burns) [inline LICENSE]: Can you do "2012 - 2015" instead?

Comment (jsom) [inline LICENSE]: Done

File: README.md [hunk 2]
```diff
@@ -123,19 +124,42 @@ class @AlbumShower
...
-We love pull requests.
+See the [CONTRIBUTING] document. Thank you, [contributors]!
 
-1. Fork the repo.
-2. Make a feature branch.
-3. Write tests.
-4. Implement the feature/bugfix.
-5. Rebase and squash.
-6. Submit the pull request.
+[CONTRIBUTING]: CONTRIBUTING.md
```
Comment (mxie) [inline README.md]: Does this need a leading `/` like for `LICENSE` below?

File: README.md [hunk 3]
```diff
@@ -123,19 +124,42 @@ class @AlbumShower
...
 ```
 
+Need Help?
+----------
+
+We offer 1-on-1 coaching. We can help you install Qt, set up Capybara, get
+started writing integration tests, and track down slow or erratic tests. [Get in
+touch].
+
+[Get in touch]: http://coaching.thoughtbot.com/rails/?utm_source=github
```
Comment (mike-burns) [inline README.md]: None of this section is still true.

File: README.md (hunk 2 above)
Comment (jsom) [inline README.md]: Thank you for keeping me consistent.

File: README.md (hunk 3 above)
Comment (jsom) [inline README.md]: Removed

**Pull request merged**